internal/v1/handler_compose_image.go

```diff
@@ -830,6 +830,24 @@ func buildCustomizations(cust *Customizations) (*composer.Customizations, error)
 		}
 	}
 
+	// we need to explicitly add 'rhcd' to the enabled services
+	// if openscap and subscription customizations are set, otherwise
+	// the insights-client doesn't register properly
+	if cust.Subscription != nil && cust.Subscription.Insights && cust.Openscap != nil {
+		if res.Services == nil {
+			res.Services = &composer.Services{
+				Enabled: &[]string{"rhcd"},
+			}
+		} else {
+			enabledServices := []string{}
+			if res.Services.Enabled != nil {
+				enabledServices = *res.Services.Enabled
+			}
+			enabledServices = append(enabledServices, "rhcd")
+			res.Services.Enabled = &enabledServices
+		}
```

Comment:
Nitpick: I think we can simplify by just filling the blanks (not sure if the referencing works on append tho):
```suggestion
		if res.Services == nil {
			res.Services = &composer.Services{}
		} 
		if res.Services.Enabled == nil {
			res.Services.Enabled = &[]string{}
		}
		res.Services.Enabled = &append(*res.Services.Enabled, "rhcd")
```

Reply:
Tho we are not checking if it's actually not there yet, before adding. Should we? :thinking:

Reply:
I mean we could. But I am almost 99% sure that none of the profiles set it... I will double check though

Reply:
`grep -r -c "sshd" ./distributions/*/oscap`
Produces no results. So I think we're fine